content/release-10-17-0/devices-10-17-0.md

```diff
@@ -197,6 +197,13 @@ mandatory <b>Software type</b> field is filled.</td>
 <td>10.16.8.0</td>
 </tr>
 
+<tr>
```

Comment:
This ticket is already in the file, please remove from here again.

Reply:
Removed the whole section